### PR TITLE
Update operator docs for Ledger Support

### PR DESCRIPTION
**Pull request by david-yan**
Updates commands to match new configuration introduced from https://github.com/oasislabs/oasis-core/pull/2339

TODO for another PR:
- Add sections for node operators solely using Ledger.

### CODE REVIEW

**Comment by ravenac95** [comment]
@peterjgilbert should we update your branch to support these changes? So we don't drift on the docs too much.

**Comment by peterjgilbert** [comment]
> @peterjgilbert should we update your branch to support these changes? So we don't drift on the docs too much.

Sure, but we should also run another test to be safe.

**Comment by peterjgilbert** [comment]
@david-yan can you update https://github.com/oasislabs/docs/blob/master/src/operators/stake-management.md as well?

File: src/operators/stake-management.md [hunk 1]
```diff
@@ -34,7 +34,17 @@ export ADDR=unix:/tmp/oasis-net-runner236357163/net-runner/network/client-0/inte
...
+and `--signer.dir $ENTITY_DIR_PATH` will be used for offline operations.
+
+The `--signer` flag is used to determine the backend used to sign transactions
+for offline operations.
+
+* For a file based signer, use `--signer file` and include the keys in
+`$ENTITY_DIR_PATH`. This is set by default, but is included in all examples to
+be explicit.
+* For a Ledger based signer, use `--signer ledger` to indicate the use of a
+Ledger device, `--signer.ledger.address` to specify the device by address, and
```
Comment (kostko) [inline src/operators/stake-management.md]: Maybe add some instructions on how to get the list of devices by address?

Comment (david-yan) [reply src/operators/stake-management.md]: Sorry for missing that! https://github.com/oasislabs/docs/pull/155

**Pull request merged**